content/product-engineering/planning.md

```diff
@@ -0,0 +1,140 @@
+# Product and engineering planning
+
+- [Product and engineering planning](#product-and-engineering-planning)
+  - [Planning timeline](#planning-timeline)
+    - [Annual planning process](#annual-planning-process)
+    - [Quarterly planning process](#quarterly-planning-process)
+  - [Sharing progress](#sharing-progress)
+    - [Status definitions](#status-definitions)
+    - [GitHub project tracker](#github-project-tracker)
+    - [OKR slides](#okr-slides)
+    - [Monthly team updates](#monthly-team-updates)
+  - [Quarterly retrospectives](#quarterly-retrospectives)
+
+## Planning timeline
+
+### Annual planning process
+
+We go through this full process in Q4 and a lightweight version of this every quarter to review and make necessary updates.
+
+| Week of quarter | Action                                                                                                                                    |
+| --------------- | ----------------------------------------------------------------------------------------------------------------------------------------- |
+| Week 2          | Draft annual vision document for next year<li>VPP/VPE draft annual vision doc                                                             |
+| Week 3          | Gather input from stakeholders<li>VPP/VPE get feedback from exec team and key stakeholders                                                |
+| Week 4          | Update annual vision <li>VPP/VPE update annual vision draft based on feedback                                                             |
+| Week 5          | Share & gather feedback <li>VPP/VPE share annual vision with Product/Eng Directors and gather feedback                                    |
+| Week 6          | Finalize annual vision document <li>VPs update and finalize based on feedback <li>Annual vision added to handbook & communicated on Slack |
```

Comment:
**issue**: The `<li>` (list item) tags in these tables are missing enclosing `<ul>``</ul>` (unordered list) tags and closing `</li>` tags, which causes rendering issues, e.g. in Firefox:

<img width="801" alt="image" src="https://user-images.githubusercontent.com/10532611/141464075-4262fc59-fbf4-43d2-b57c-a35570876817.png">

```suggestion
| Week 6          | Finalize annual vision document <ul> <li>VPs update and finalize based on feedback</li> <li>Annual vision added to handbook & communicated on Slack</li> </ul> |
```

(needs to be done for all `<li>`s in this PR)